projects/packages/woocommerce-analytics/src/class-woo-analytics-trait.php

```diff
@@ -264,19 +286,15 @@ public function find_cart_checkout_content_sources() {
 	 * @return array Array of standard event props.
 	 */
 	public function get_common_properties() {
-		// phpcs:disable Squiz.PHP.CommentedOutCode.Found
-		// Disabled the below temporarily to avoid caching issues.
-		// $session_data       = json_decode( sanitize_text_field( wp_unslash( $_COOKIE['woocommerceanalytics_session'] ?? '' ) ), true ) ?? array();
-		// $session_id         = sanitize_text_field( $session_data['session_id'] ?? $this->session_id );
-		// $landing_page       = sanitize_url( $session_data['landing_page'] ?? $this->landing_page );
-		// phpcs:enable Squiz.PHP.CommentedOutCode.Found
+		$session_id         = $this->get_session_id();
+		$landing_page       = $this->get_landing_page();
 		$site_info          = array(
-			'session_id'                         => null,
+			'session_id'                         => $session_id,
 			'blog_id'                            => Jetpack_Connection::get_site_id(),
 			'store_id'                           => defined( '\\WC_Install::STORE_ID_OPTION' ) ? get_option( \WC_Install::STORE_ID_OPTION ) : false,
 			'ui'                                 => $this->get_user_id(),
-			'url'                                => home_url(),
-			'landing_page'                       => null,
+			'url'                                => $this->get_current_url(),
```

Comment:
In https://github.com/Automattic/jetpack/pull/41085, we updated the `url` property to use the actual page URL instead of `home_url()`. Since this could be a breaking change, we should check with the Data team to make sure this update doesn't break anything. As for WA widgets (hdK9XLiOefugy2bxalKBN1-fi-6015_329167), none of them require the URL property to display correctly.

Reply:
I've undone the URL change in commit e6ecf544bb718b26c1c436670ba037bc1e51479a as it could break the existing reporting. We should implement an approach to ensure the full page URL is only sent when transmitting data to ClickHouse.